README.md

```diff
@@ -28,6 +28,43 @@ In development, create an admin superuser
 nix-shell --run "./manage.py createsuperuser"
 ```
 
+### Setup Koala connection
+
+(In development) Change `KOALA_DB_NAME` to `koala-development`
+
+Create the `undead_mongoose` user in Koala's database:
+```sql
+CREATE USER undead_mongoose WITH PASSWORD 'mongoose123';
+```
+
+Configure privileges for the `undead_mongoose` user in Koala's database (Replace `koala` with `koala-development` in development):
+```sql
+\c "koala"
+GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA public TO undead_mongoose;
+```
+
+Setup OAuth2:
+In Koala's Database (Replace `koala` with `koala-development` in development):
+```sql
+\c "koala"
+INSERT INTO public.oauth_applications(name, uid, secret, redirect_uri) VALUES ('mongoose', 'example_id', 'example_secret', 'http://localhost:8000/oidc/callback/');
```

Comment:
Even though this is not wrong it does not limit the scopes and a preferable way would probably.
running this command in a nix-shell of koala
`bundle exec rake "doorkeeper:create[undead Mongoose, http://localhost:8000/oidc/callback/, openid profile email member-read]"`
Granting undead_mongoose access to koala should not be required and probably a bad idea